nmkapp/pipeline.py

```diff
@@ -19,3 +20,11 @@ def request_display_name(strategy, details, user=None, is_new=False, *args, **kw
         return
     else:
         return redirect('request_display_name')
+
+
+@transaction.atomic
```

Comment:
cemu sluzi ovaj transaction.atomic ovde? (pitam zbog sebe)

Reply:
Upisujem u bazu, mada koristi samo jedan query cini mi se, tako da je mozda visak. Stavio sam ga cim sam pravio pipeline

Reply:
ali to je deo neke druge transakcije, ne vidim zasto bi zeleo na pozivu ove funkcije da pocinjes novu (nested) transakciju, nema mi smisla. Ja bih to uklonio:/